test/t/test_sudo.py

```diff
@@ -81,3 +82,13 @@ def test_11(self, bash, part_full_group):
         part, _ = part_full_group
         completion = assert_complete(bash, "sudo chown foo:bar:%s" % part)
         assert not completion
+
+    def test_12(self, bash):
+        completion = assert_complete(bash, 'sudo "/tmp/aaa bbb" ')
```

Comment:
The variable `completion` is unused.
```suggestion
        assert_complete(bash, 'sudo "/tmp/aaa bbb" ')
```